src/core/test/lib/helpers.ts

```diff
@@ -19,7 +19,6 @@ export async function assertRequest(path: string, method: HttpVerb, expected: Ob
         assert.fail(`Threw error making request to and parsing body from ${method.toUpperCase()} ${path}.`);
     }
     assert.deepEqual<Object>(
-        // @ts-ignore
```

Comment:
Also a little bit out of scope, but since the line below was not throwing an error I removed the @ts-ignore comment.